sklearn/ensemble/_bagging.py

```diff
@@ -227,11 +227,11 @@ def fit(self, X, y, sample_weight=None):
             The training input samples. Sparse matrices are accepted only if
             they are supported by the base estimator.
 
-        y : array-like of shape (n_samples,)
+        y : ndarray of shape (n_samples,)
```

Comment:
For all of the input let's keep array-like. If y is a list it will still work, it doesn't need to be an ndarray.